Classes/Controller/PageViewController.php

```diff
@@ -312,7 +312,7 @@ public function addDocumentAction(FormAddDocument $formAddDocument): ResponseInt
      * @param int|null $docNumber
      * @return array
      */
-    protected function getMeasures(int $page, MetsDocument $specificDoc = null, $docNumber = null): array
+    protected function getMeasures(int $page, ?MetsDocument $specificDoc = null, $docNumber = null): array
```

Comment:
Does function also with `int` or `string`?

If yes then, `?int $docNumber` could be added here.

Reply:
> Does function also with `int` or `string`?

Yes, see for example https://www.php.net/manual/en/migration71.new-features.php.

> If yes then, ?int $docNumber could be added here.

That's right.